cpu/cc26x0/periph/i2c.c

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (C) 2018 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_cc26x0
+ * @ingroup     drivers_periph_i2c
+ * @{
+ *
+ * @file
+ * @brief       Low-level I2C driver implementation
+ * @note        This CPU has weak pullups, external pullup resistors may be
+ *              required.
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * @}
+ */
+
+#include <errno.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "mutex.h"
+
+#include "cpu.h"
+#include "periph/i2c.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
+
+#define PREG(x) DEBUG("%s=0x%08x\n", #x, (unsigned)x);
+
+/**
+ * @brief Array holding one pre-initialized mutex for each I2C device
```

Comment:
its not an array anymore also for the note below: how should there be multiple devices if the MCU only has one?